rasa/core/events/__init__.py

```diff
@@ -989,11 +996,17 @@ def __init__(
         confidence: Optional[float] = None,
         timestamp: Optional[float] = None,
         metadata: Optional[Dict] = None,
-    ):
+        # TODO: Throw out message attribute
+        message: Optional[Message] = None,
```

Comment:
@evgeniiaraz This todo was just for the `e2e-wip-parser` branch. We don't need to carry it over.

Reply:
I wanted to leave it in until I add the new core featurization for some sanity checks there and then remove it.

Reply:
let's treat these PRs as PRs against master, therefore there is no reason to introduce smth to remove it later